pkg/linters/components/pkgconfigusage/go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/DataDog/datadog-agent/pkg/linters/components/pkgconfigusage
+
+go 1.22.4
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Why Go 1.22.4 ? Is this the version used to build golangci-lint ?
Can we use 1.22.0 instead like in every other module ?
When does this need to be updated, when bumping golangci-lint or when bumping Go ?